tardis/montecarlo/src/rpacket.c

```diff
@@ -15,9 +15,11 @@ rpacket_init (rpacket_t * packet, storage_model_t * storage, int packet_index,
   double current_nu = storage->packet_nus[packet_index];
   double current_energy = storage->packet_energies[packet_index];
   double current_mu = storage->packet_mus[packet_index];
-  double comov_current_nu = current_nu;
+//  double comov_current_nu = current_nu;
   int current_shell_id = 0;
   double current_r = storage->r_inner[0];
+  double comov_current_nu = current_nu * (1 - (current_mu * current_r * storage->inverse_time_explosion * INVERSE_C)); 
+/*  
```

Comment:
Why did you change this? If this is really a bug, it is disjunct from the formal integral and should be fixed in separate patch (i.e. PR)